.buildkite/pipeline.release-experimental.yml

```diff
@@ -32,6 +32,21 @@ steps:
           volumes:
             - "/yum.buildkite.com"
 
+  - name: ":redhat: Publish Edge RPM Package to Packagecloud"
+    command: ".buildkite/steps/publish-rpm-packagecloud.sh"
+    env:
+      REPOSITORY: "buildkite/agent-experimental"
+      DISTRO_VERSION: rpm_any/rpm_any
+    agents:
+      queue: "deploy"
+    plugins:
+      - docker#v5.8.0:
+          image: "buildkite/agent:3.55.0-ubuntu"
+          entrypoint: bash
+          propagate-environment: true
+          mount-buildkite-agent: true
```

Comment:
could this just be a `ruby` container with `mount-buildkite-agent` turned on?

Reply:
or just an ubuntu container (which have ruby installed by default i guess?)

Reply:
Maybe! I've picked the same as the other step because it's already known to work, and because it's likely to already be in cache. Feel free to modify it, if you modify the other steps :pray:

Reply:
(Eventually I'd like us to have a Golang single-binary CLI for packages, and to be able to distribute that as part of a binary plugin instead of using docker!)